routes/reminders.js

```diff
@@ -52,9 +59,16 @@ router.route('/:id')
       .catch(next);
   })
   .put((req, res, next) => {
-    reminders.update(req.user.family, req.params.id, req.body)
-      .then(() => res.status(204).end())
-      .catch(next);
+    const family = req.user.family;
+    const id = req.params.id;
+
+    reminders.update(family, id, req.body).then(() => {
+      debug('Reminder #%s has been updated in database', id);
+
+      return reminders.show(family, id);
+    }).then((reminder) => {
```

Comment:
What do you think of debugging the reminder returned here? https://github.com/fxbox/calendar-server/pull/20/files#diff-684d58cfa73d5e1d3d973792c90a623dR58